boards/arm/nrf52840_pca20041/nrf52840_pca20041.dts

```diff
@@ -128,4 +128,9 @@
 &usbd {
 	compatible = "nordic,nrf-usbd";
 	status = "ok";
+	num-bidir-endpoints = <0>;
```

Comment:
Why are you reducing it for the whole board? Cannot it be just limited in the example?

Reply:
This is our board (nrf_desktop mouse). nrf_desktop is a reference design not an example and board is not intended to be used with examples. Clients will create their own board definitions based on ours.
I doubt this board will be used for anything else beside our project (see sr3 boards). It is located in the common directory only because of the unification reasons. Board has everything configured only for our project. We could create an overlay, but what for...

Reply:
To keep an order. Ones may base on this board definition. If this is limited version it should not contain nrf52840 in the name. Seems like dirty hack.

Reply:
I thought about it again when adding the same change for dongle and decided to use an overlay so we have change in the same place for both boards.